From: Steve Dickson <steved@redhat.com>
Date: Thu, 29 Jun 2017 11:48:26 -0400
Subject: [PATCH] mount: copy the port field into the cloned nfs_server
 structure.
Git-commit: 89a6814d9b665b196aa3a102f96b6dc7e8cb669e
Patch-mainline: v4.13
References: bsc#1136990

Doing this copy eliminates the "port=0" entry in
the /proc/mounts entries

Fixes: https://bugzilla.kernel.org/show_bug.cgi?id=69241

Signed-off-by: Steve Dickson <steved@redhat.com>
Signed-off-by: Anna Schumaker <Anna.Schumaker@Netapp.com>
Acked-by: NeilBrown <neilb@suse.com>

---
 fs/nfs/client.c |    1 +
 1 file changed, 1 insertion(+)

--- a/fs/nfs/client.c
+++ b/fs/nfs/client.c
@@ -835,6 +835,7 @@ void nfs_server_copy_userdata(struct nfs
 	target->caps = source->caps;
 	target->options = source->options;
 	target->auth_info = source->auth_info;
+	target->port = source->port;
 }
 EXPORT_SYMBOL_GPL(nfs_server_copy_userdata);
 
